src/App.jsx

```diff
@@ -14,33 +16,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const nameSort = 'name';
+const lengthSort = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const sortGoods = (goods, { sort, isReverse }) => {
+  const sortGood = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sort) {
+    sortGood.sort((good1, good2) => {
+      switch (sort) {
+        case nameSort: {
+          return good1.localeCompare(good2);
+        }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+        case lengthSort: {
+          return good1.length - good2.length;
+        }
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  return isReverse ? sortGood.reverse() : sortGood;
+};
```

Comment:
The `sortGoods` function is implemented correctly, but consider renaming `sortGood` to `sortedGoods` for better readability, as it represents the sorted array of goods.